SU-SWS/saa_victoria_subtheme
# ADAPT-665: Removed extra footer link styles and checked big button in…

… in-page banner.

# READY

# Summary
- Remove redundant code in footer links
- remove old code with big button class

# Review By (Date)
- 9/28

# Urgency
- low - good practice code cleanup

# Steps to Test

1. Pull down the branch or look at https://saauserguide.sites.stanford.edu/site-building-guide/saa-components-stanford-sites
2. Clear the site cache
3. Test that footer links all look correct with colors and hover states
4. check that big button styles work inside in-page red banner (no image banner)  - edit an in-page red banner, change button to big button class style, check, and change back on page https://saauserguide.sites.stanford.edu/site-building-guide/saa-components-stanford-sites/page-banners or **on an unpublished test page**.
5. check that all banner types look the same / havent changed (2 in page types and top of page)
- in page has with image and without image = 2 very different styles
https://saauserguide.sites.stanford.edu/live-example-top-banner-only-text
https://saauserguide.sites.stanford.edu/top-banner-both-text-and-image
https://saauserguide.sites.stanford.edu/site-building-guide/saa-components-stanford-sites/page-banners

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-665
